### PR TITLE
vassalengine.VASSAL version 3.5.8.0

### PR DESCRIPTION
**Pull request by jeandeaual**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28042)

### CODE REVIEW

**Review by quhxl — CHANGES_REQUESTED** [review]
`Publisher` and `Version` must be equal to the entry in the control panel.

image

Don't forget to rename the folder when you adjust the version number.

File: manifests/v/vassalengine/VASSAL/3.5.8.0/vassalengine.VASSAL.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: vassalengine.VASSAL
+PackageVersion: 3.5.8.0
```
Comment (ItzLevvie) [inline manifests/v/vassalengine/VASSAL/3.5.8.0/vassalengine.VASSAL.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.5.8
```

File: manifests/v/vassalengine/VASSAL/3.5.8.0/vassalengine.VASSAL.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: vassalengine.VASSAL
+PackageVersion: 3.5.8.0
```
Comment (ItzLevvie) [inline manifests/v/vassalengine/VASSAL/3.5.8.0/vassalengine.VASSAL.installer.yaml:5]:
```suggestion
PackageVersion: 3.5.8
```

File: manifests/v/vassalengine/VASSAL/3.5.8.0/vassalengine.VASSAL.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: vassalengine.VASSAL
+PackageVersion: 3.5.8.0
+PackageLocale: en-US
+Publisher: vassalengine
```
Comment (ItzLevvie) [inline manifests/v/vassalengine/VASSAL/3.5.8.0/vassalengine.VASSAL.locale.en-US.yaml:7]:
```suggestion
Publisher: vassalengine.org
```

File: manifests/v/vassalengine/VASSAL/3.5.8.0/vassalengine.VASSAL.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: vassalengine.VASSAL
+PackageVersion: 3.5.8.0
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/vassalengine/vassal/releases/download/3.5.8/VASSAL-3.5.8-windows-32.exe
+  InstallerSha256: 762A7BFE78C58CCDE7C1C3E9CAC812E8743E398E248ADFC4E778FFAD82D710A6
```
Comment (ItzLevvie) [inline manifests/v/vassalengine/VASSAL/3.5.8.0/vassalengine.VASSAL.installer.yaml:10]:
```suggestion
  InstallerSha256: 762A7BFE78C58CCDE7C1C3E9CAC812E8743E398E248ADFC4E778FFAD82D710A6
  Scope: machine
```

File: manifests/v/vassalengine/VASSAL/3.5.8.0/vassalengine.VASSAL.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+PackageVersion: 3.5.8.0
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/vassalengine/vassal/releases/download/3.5.8/VASSAL-3.5.8-windows-32.exe
+  InstallerSha256: 762A7BFE78C58CCDE7C1C3E9CAC812E8743E398E248ADFC4E778FFAD82D710A6
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/vassalengine/vassal/releases/download/3.5.8/VASSAL-3.5.8-windows-64.exe
+  InstallerSha256: D4FB3777AE768717D9C0BC0CAF92171B28155E1A19E244C06FDE9743D37ED1D8
```
Comment (ItzLevvie) [inline manifests/v/vassalengine/VASSAL/3.5.8.0/vassalengine.VASSAL.installer.yaml:14]:
```suggestion
  InstallerSha256: D4FB3777AE768717D9C0BC0CAF92171B28155E1A19E244C06FDE9743D37ED1D8
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/28042#pullrequestreview-758136353

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**